hardhat.config.ts

```diff
@@ -41,38 +55,21 @@ const hhconfig: HardhatUserConfig = {
       accounts: [process.env.DEPLOYER_PRIVATE_KEY || "0x0"],
       gasPrice: 30000000, // this is 30 Gwei
     },
+    base_testnet: {
+      url: process.env.BASE_TESTNET_RPC_URL,
+      accounts: [process.env.DEPLOYER_PRIVATE_KEY || "0x0"],
+      gasPrice: 30000000, // this is 30 Gwei
+    },
     celo: {
       url: process.env.CELO_RPC_URL,
       accounts: [process.env.DEPLOYER_PRIVATE_KEY || "0x0"],
       gasPrice: 5000000000, // this is 30 Gwei
     },
-  },
-  etherscan: {
```

Comment:
this is needed for contract verification

Reply:
It returns an error because you use typescript and `etherscan` is not part of the definition of the ``hhconfig` type.

So it should be stored somewhere else. 

(where is it used? I did a global search and couldn't find where it is used, that's why I simply removed it, thinking it was a legacy thing)

Reply:
it is valid. I think that you have an issue with your ts setup on your machine. Even the deploy.ts file is filled with typescript errors.

Reply:
Typings are there and come through `@nomicfoundation/hardhat-toolbox`

![Screenshot 2023-11-30 at 10 02 43 (2)](https://github.com/citizenwallet/smartcontracts/assets/122159642/ef15f58a-4da0-41d6-9957-0b430dc5d6b2)

Reply:
it's used where you see `await run("verify:verify"` in `deploy-community.ts`.

Hardhat will automatically verify with etherscan using that command.